Lib/idlelib/editor.py

```diff
@@ -1203,10 +1203,7 @@ def load_extension(self, name):
             self.apply_bindings(keydefs)
             for vevent in keydefs:
                 methodname = vevent.replace("-", "_")
-                while methodname[:1] == '<':
-                    methodname = methodname[1:]
-                while methodname[-1:] == '>':
-                    methodname = methodname[:-1]
+                methodname = methodname.lstrip('<').rstrip('>')
```

Comment:
The line above and below can be merged into this line to produce this 1 line
`                methodname = vevent.lstrip("<").replace("-", "_").rstrip(">") + "_event"`
that replaces 6 (1205 to 1210).